loopy/target/ispc.py

```diff
@@ -392,7 +392,8 @@ def emit_assignment(self, codegen_state, insn):
 
         # {{{ handle streaming stores
 
-        if "!streaming_store" in insn.tags:
+        from loopy.kernel.instruction import UseStreamingStoreTag
+        if UseStreamingStoreTag() in insn.tags:
```

Comment:
Should Non-ISPC targets raise if they see an instruction tagged `UseStreamingStore`?

Reply:
Good question. I view the tag as advisory. I'll add a note.

Reply:
1b3e1380

Reply:
19c74e5a9deb2cbfe6e99126a020cf53508c09bc